### ISSUE TITLE
Gerber file /pcb layout 

### ISSUE THREAD

**Issue by Amp208** [Proto-App/Proto-Android#315]
Just wonder if you guys were adding a gerber file option or pcb design app where i can transfer my circuit designs to a pcb layout.

**Comment by Proto-App** [comment]
Yes, we would like to create such feature. But we have features before.
